packages/frint-router-react/README.md

```diff
@@ -0,0 +1,333 @@
+# frint-router-react
+
+[![npm](https://img.shields.io/npm/v/frint-router-react.svg)](https://www.npmjs.com/package/frint-router-react)
+
+> Router package for Frint
+
+<!-- MarkdownTOC autolink=true bracket=round -->
+
+- [Guide](#guide)
+  - [Installation](#installation)
+  - [Usage](#usage)
+  - [Nested routes](#nested-routes)
+  - [Note](#note)
+- [API](#api)
+  - [Route](#route)
+  - [Link](#link)
+  - [Switch](#switch)
+
+<!-- /MarkdownTOC -->
+
+---
+
+# Guide
+
+## Installation
+
+With [npm](https://www.npmjs.com/):
+
+```
+$ npm install --save frint-router frint-router-react
+```
+
+Via [unpkg](https://unpkg.com) CDN:
+
+```html
+<script src="https://unpkg.com/frint-router-react@latest/dist/frint-router-react.min.js"></script>
+
+<script>
+  // available as `window.FrintRouterReact`
+</script>
+```
+
+## Usage
+
+This package exports a handful of components.
+
+### `Route` component
+
+The `Route` component is how you define a route in your application, that can render either a Component or an App.
+
+#### Components
+
+Assume you have two components `HomePage` and `AboutPage`, and you want to show them when the browser navigates to `/` and `/about` respectively:
+
+```js
+// components/Root.js
+import { Route } from 'frint-router-react';
+
+import HomePage from './HomePage';
+import AboutPage from './AboutPage';
+
+export default function Root() {
+  return (
+    <div>
+      <Route path="/" component={HomePage} exact />
+      <Route path="/about" component={AboutPage} />
+    </div>
```

Comment:
it will render, once the current location in the browser matches to any of the paths. if there is a match, then the assigned component (`HomePage` or `AboutPage`) will render accodingly.